spec/features/user/user_can_remove_items_from_cart_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.feature 'user removes items from cart' do
+
+  let!(:user) { create(:user) }
+  let!(:category) { create(:category) }
+  let!(:category) { create(:category) }
+  let!(:item) { create(:item) }
+
+  scenario 'as a logged in user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.